data/hai/hai reveal 0 prologue.txt

```diff
@@ -0,0 +1,736 @@
+			`	Sayari chuckles. "So much on that human's mind. I have not seen him take a break since he has arrived. It is as if he is more stressed about this situation than any Hai."`
+				decline
+
+
+
+mission "Hai Reveal: Meet the Team: Xilin"
+	name "New Arrivals"
+	description "Travel to <stopovers> with Xilin to interview new arrivals to Hai space to determine how they arrived and what they think about the Hai. Return to <destination> when finished."
+	source "Hai-home"
+	stopover "Allhome"
+	passengers 2
+	blocked "You will need <capacity> in order to take this mission."
+	to offer
+		has "hr: meet the team"
+		has "Hai Reveal: Meet the Team: Alondo: done"
+	on offer
+		log "People" "Xilin Yang" `Deputy Minister Yang works in the Foreign Affairs department of the Republic government. He is a pleasant and astute man. Handling the Hai secrecy crisis has become his job while the majority of the foreign ministry remains engaged with the fallout from the Free Worlds and recent peace.`
+		conversation
+			`Sayari follows you out to the landing pads to meet the presenter. "Captain <last>," he says while grasping your hand in a firm handshake. "Xilin Yang, Republic Deputy Minister of Foreign Affairs. I apologize for not introducing myself sooner. It's an honor to meet a hero of the Pug crisis such as yourself."`
+			choice
+				`	"It was hard-fought, but it was necessary."`
+				`	"Thanks, but it wasn't as dramatic as they made it out to be. I'm just happy to have helped where I could."`
+					goto humble
+			`	"You're every inch what I expected," he assures you.`
+				goto mission
+			label humble
+			`	For a moment his expression falters, and then he seems almost relieved. "You may be a better person than I expected to meet.`
+			label mission
+			`	"Now, about that task. I mentioned during our briefing that we need to determine how much of a threat these new arrivals might be. Not a threat in the sense that they will become violent, but in the sense that they could exacerbate the situation, such as by returning to our space and making a big stink about what they've found here. The best place to go for this would be <stopovers>. I have a sentry there keeping watch and recording new arrivals, but I'd like to meet some of them for myself."`
+			`	"I would like to follow and observe," says Sayari.`
+			`	"You're free to come," says Xilin. "Let me pack a few items into your ship, Captain <last>, and we can be on our way."`
+				accept
+	on stopover
+		conversation
+			`You meet with the sentry that Xilin mentioned back on <planet>, who turns out to be a Republic Intelligence officer who's been recording pilot's licenses and ship registration of all new arrivals. He hands over a data stick of notes to Xilin. "They make themselves very obvious with the way they gawk," he says. "We had two new ships come in yesterday, a Freighter and an Osprey, but there's been nothing today as of yet. It's normally only one or two smaller ships a day, if that."`
```

Comment:
So the thing about there being a Sentry:
1. If they're an established entity there's nothing to stop Xilin simply calling them, for the majority of updates - or having someone else in his employ do a collection run. He could also do a simple collection trip himself at a pinch, because he doesn't need a captain who can facilitate an uncertain schedule of investigating and such.
2. If there's an actual republic agent, they're presumably competent enough (and in less physical danger) to be confronting new captains arriving, which completely negates the value of Xilin wandering around himself.
3. The presence of a republic agent publicly situated in Hai space would _have_ to have acquired some sort of Hai approval, possibly some sort of Hai oversight, and likely some sort shared reporting arrangement with the Hai. This is the nature of intergovernmental relations involving the public deployment of government agents - which means that Sayari would have no need to accompany them to find out anything because she'd have had to have been in the loop for setting it up in the first place.
4. (3.b) If the agent acting as Sentry was a secret agent, then this could work, but then they cannot be confronting pilots publicly, and then Xilin would absolutely not want Sayari to be coming along.

This was why the original suggestion had you _discover_ someone who has every right to be there already - who is simply conscientiously willing to help. A citizen informer doesn't have any of the baggage that an agent does, and cannot be leverage by the Hai government to extract anything from the Republic.

Note: This is not to say that Xilin does/would _not_ have an agent around, he might. A hypothetical agent might even have passed on information about the captain doing the recording so that Xilin could conveniently "find" him.
A public and formally employed person though, creates a number of holes that would need patching.

Reply:
Then how about just taking the captain that was mentioned at the end and moving them to the front here?
```suggestion
			`	"I would like to follow and observe," says Sayari.`
			`	"You're free to come," says Xilin. "Let me pack a few items into your ship, Captain <last>, and we can be on our way."`
				accept
	on stopover
		conversation
			`The rate at which new ships show up isn't tremendous, so Xilin starts by speak with some of the older captains who have been here prior to the news. "You can learn a lot by sharing a drink with the right merchant," he says to you.`
			`	One captain in particular has been camped out here to intercept new captains as they arrive and give them the induction tour. He also has a copious set of notes that he gladly shares with Xilin. "After I heard the news, I figured I'd help by keeping notes on new arrivals. They make themselves very obvious with the way they gawk," he says. "We had two new ships come in yesterday, a Freighter and an Osprey, but there's been nothing today as of yet. It's normally only one or two smaller ships a day, if that."`
```